pkg/caretta/links_tracer.go

```diff
@@ -46,7 +45,7 @@ type Probes interface {
 
 type LinksTracer struct {
 	ebpfObjects Probes
-	connections *ebpf.Map
+	connections ConnectionsMap
```

Comment:
Maybe rename the interface to eBPFMap? It's specifically designed for that usecase

Reply:
I already have a type EbpfMap which is the wrapper for ebpf.Map
should I rename it to EbpfMapWrapper? or maybe GoEbpfMap?

Reply:
IEbpfMap is also possible :)